es6.md

```diff
@@ -30,6 +30,16 @@ extend interface VariableDeclaration {
 }
 ```
 
+## HoistableDeclaration
```

Comment:
@sebmck @caridy I forgot where we discussed adding the HoistableDeclaration node. Did @michaelficarra or @ariya have issues there? I forgot...